conjure/steps/01_get-kubectl.sh

```diff
@@ -20,7 +20,7 @@ if [ $(unitStatus kubernetes 0) != "active" ]; then
 fi
 
 # TODO: Convert to an actionable item
-juju scp kubernetes/0:kubectl_package.tar $KUBECTL_PATH/.
-cd $KUBECTL_PATH && tar zxf kubectl_package.tar
+juju scp kubernetes/0:kubectl_package.tar.gz $KUBECTL_PATH/.
```

Comment:
The master is not always node 0. We should come up with a way to find the master unit for any given charm (even if it is some contrived juju run command that parses the result).